pyke/lightcurve.py

```diff
@@ -1067,3 +1067,63 @@ def _pld(self, tpf_flux, polyorder=2):
         model = np.dot(X, opt_weights)
         flux_hat = lightcurve - model
         return flux_hat
+
+
+class BoxLikePeriodSearch(object):
+    def __init__(self):
+        pass
+
+    def search(self, lc, minper, maxper, nperiods=2000, prior=None):
+        """
+        Parameters
+        ----------
+        lc : LightCurve object
+            An object from KeplerLightCurve or LightCurve
+        minper : float
+            Minimum period to search for.
```

Comment:
You could say that the units must be the same as the units of `lc.time`.

Reply:
👍